cmd/mailchain/internal/http/handlers/messages.go

```diff
@@ -206,6 +211,14 @@ type getMessage struct {
 	// readOnly: true
 	// example: true
 	Read bool `json:"read"`
+
+	BlockId []byte `json:"block-id"`
```

Comment:
I should of been more clear on this before `BlockId string `json:"block-id"` is prefered. As if its `[]byte` the output is difficult to read if a human is looking at the json.